### ISSUE TITLE
Run Triples without requiring NVidia Surround

### ISSUE THREAD

**Issue by brettswift** [terrymacdonald/DisplayMagician#229]
This is an idea to run triples without NVidia Surround, similar to what SRWE.exe does (Simple Runtime Window Editor). 

SRWE.exe:  [https://github.com/dtgDTGdtg/SRWE](https://github.com/dtgDTGdtg/SRWE)

I don't like running NVidia Surround, as in full screen I can't tab out and back and have the game behave normally.   My current workflow is to: 

1. run ACC
2. Run SRWE.exe
  a. Select AC2 from the running programs 
  b. load my profile (that tells AC2 to stretch to triple monitors)

**Describe the solution you'd like**
It would be nice to have DisplayMagician take a profile for a game and stretch it's boundaries to the triple monitors that I have. 

**Describe alternatives you've considered**
running SRWE.exe with arguments, however that program is out of support for a couple of years and doesn't accept arguments. (it seems?)

Great app,  thanks for putting work into this!

**Comment by terrymacdonald** [comment]
Hi Brett,

I took a quick look at this over the weekend, and it's not something that
can easily be added to DisplayMagician. DisplayMagician focuses on
storing and applying configurations from Video Card drivers. What SRWE does
is move and modify windows themselves, and that's a whole different
ballgame. I think that would just complicate DisplayMagicians codebase, and
make it harder to support the program.

What would make more sense, is to add command line functionality to SRWE.
I've had a quick look at the codebase, and I think it would be possible to
do so with some work. I'll add it to my backlog of things to do, and
hopefully I'll find some spare time within the next 6 months or so. Sorry I
can't give any other more concrete outcomes but life has gotten rather
hectic of late and I don't have as much time to programme as I used to.

Thanks
*Terry *
